### DISCUSSION TITLE
Duplicate inactive relationship is not flagged as error in UI.

### DISCUSSION THREAD

**Discussion by atmisra** [TabularEditor/TabularEditor3#1092] [Ideas]
Unlike duplicate _active_ relationships (which have a red error indicator when duplicated), duplicate _inactive_ relationships do not show an error in the UI. Error is presented only on save/deployment.

**Comment by otykier** [comment]
Thanks for letting us know.

In general, there are many modelling misconfigurations that could cause an AS error upon Save/Deployment. We are not currently able to guarantee that we can catch all of these errors offline, but will add this specific issue to the backlog.
